packages/synapse-react-client/src/testutils/ReactQueryMockUtils.ts

```diff
@@ -27,7 +28,7 @@ export function getUseQuerySuccessMock<TData>(
     isRefetching: false,
     isStale: false,
     isSuccess: true,
-    refetch: jest.fn(),
+    refetch: noop as any,
```

Comment:
Cannot use these helpers that contain `jest.fn()` in a Vitest module. Might need to find a way to pass `jest` or `vitest` as a parameter, but for now, just use Lodash noop